gradle-suppressible-error-prone/src/test/groovy/com/palantir/gradle/suppressibleerrorprone/AbstractSuppressibleErrorPronePluginIntegrationTest.groovy

```diff
@@ -0,0 +1,129 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.suppressibleerrorprone
+
+
+import nebula.test.IntegrationSpec
+import nebula.test.functional.ExecutionResult
+import org.apache.commons.io.FileUtils
+
+class AbstractSuppressibleErrorPronePluginIntegrationTest extends IntegrationSpec {
```

Comment:
I separated out the tests into two sets:
1. Tests that interact with the gradle errorprone options
2. Checked-in style tests that test how suppressing works